unstructured/partition/text.py

```diff
@@ -261,11 +261,13 @@ def element_from_text(
     coordinate_system: Optional[CoordinateSystem] = None,
 ) -> Element:
     if is_bulleted_text(text):
-        return ListItem(
-            text=clean_bullets(text),
-            coordinates=coordinates,
-            coordinate_system=coordinate_system,
-        )
+        clean_text = clean_bullets(text)
+        if clean_text:
+            return ListItem(
+                text=clean_bullets(clean_text),
```

Comment:
```suggestion
                text=clean_text,
```
don't need to `clean_text` again?

Reply:
Yep that's what I meant! Thanks